src/igvfd/schemas/tissue.json

```diff
@@ -51,22 +50,11 @@
     "dependentSchemas": {
         "$merge": [
             "biosample.json#/dependentSchemas"
-        ],
-        "ccf_id": {
-            "comment": "Common coordinate framework identifier only applies to human tissue samples.",
-            "required": [
-                "taxa"
-            ],
-            "properties": {
-                "taxa": {
-                    "const": "Homo sapiens"
-                }
-            }
-        }
+        ]
```

Comment:
i think you should write a ticket that requires addition of an audit to flag CCF that si not human sample